src/java/org/apache/cassandra/db/SystemKeyspace.java

```diff
@@ -1409,7 +1413,7 @@ public static void snapshotOnVersionChange() throws IOException
                                                                              previous,
                                                                              next));
             for (String keyspace : SchemaConstants.LOCAL_SYSTEM_KEYSPACE_NAMES)
-                Keyspace.open(keyspace).snapshot(snapshotName, null);
+                Keyspace.open(keyspace).snapshot(snapshotName, null, false, snapshotRateLimiter);
```

Comment:
should system keyspace do got a snapshot rate limiter ? System keyspace will not got many sstables. for example, if we set the rate to 2 this may affect the speed of setup for the snapshotOnVersionChange , But the data keyspace's snapshot speed of 2 is what we want them to be .

Reply:
This is addressed.